drizzlepac/hlautils/catalog_utils.py

```diff
@@ -920,7 +917,8 @@ def identify_sources(self, **pars):
                 else:
                     bad_segm_rows_by_id.append(total_measurements_table['id'][i])
             updated_table = Table(rows=good_rows, names=total_measurements_table.colnames)
-            # log.info("SEGMENT. Bad total rows: {}".format(bad_segm_rows_by_id))
+            if self.debug:
+                log.info("SEGMENT. Bad total rows: {}".format(bad_segm_rows_by_id))
```

Comment:
If this is a statement that only should be reported in debug mode, I would think it would be simplest to convert `log.info` to `log.debug` and remove the if statement.